schema.graphql

```diff
@@ -20,25 +20,25 @@ type Sender @entity {
 
 type Receiver @entity {
   # receiver address
-  id: ID!
+  id: Bytes!
   escrowAccounts: [EscrowAccount!] @derivedFrom(field: "receiver")
   transactions: [Transaction!] @derivedFrom(field: "receiver")
 }
 
 type Transaction @entity{
```

Comment:
From the recommendations, you can also set `Transaction` as an immutable entity.

Reply:
Oh you are right! I did read it and thought none was inmutable but yeah transaction is